src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TIME = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const LONG_TERM = 7;
```

Comment:
```suggestion
  const SHORT_TERM = 3;
  const SHORT_TERM_DISCOUNT = 20;

  const LONG_TERM = 7;
```
I think it's better to follow the same naming)